apps/condo/domains/news/access/GetNewsSharingRecipientsService.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Generated by `createservice news.GetNewsItemSharingRecipientsService --type queries`
+ */
+const get = require('lodash/get')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+
+async function canGetNewsSharingRecipients ({ authentication: { item: user }, args: { data: { b2bAppContext: { id: b2bAppContextId } } } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    const b2bContext = await getById('B2BAppContext', b2bAppContextId)
+    if (!b2bContext) { return false }
+
+    const organizationId = get(b2bContext, 'organization', null)
+    if (!organizationId) { return false }
+
+    return await checkPermissionInUserOrganizationOrRelatedOrganization(user.id, organizationId, 'canReadNewsItems')
```

Comment:
Why `canReadNewsItems`?

Reply:
It was made intentionally. If you can read and manage newsItems, then you can send news to telegram and other services

Reply:
> ...you can read and manage newsItems, then you can send news to telegram and other services

What about to check `canManage` permission?

Reply:
will double check it

Reply:
Got it, changed permission to canManageNewsItems. Thanks!